docs/quickstart.rst

```diff
@@ -1,14 +1,48 @@
 Quick start
 ===========
 
-Install using `nodejs <http://nodejs.org>`_::
+Installation 
+------------
+
+First, install `git <https://git-scm.com/downloads>`_. 
+
+Second, install `Node.js <http://nodejs.org>`_. Webppl is written in JavaScript, and requires Node to run. After it is installed, you can use npm (**n**\ ode **p**\ ackage **m**\ anager) to install webppl::
 
     npm install -g webppl
 
-Run webppl programs::
+Create a file called ``test.wppl``::
+
+    var greeting = function () {
+        return flip(.5) ? "Hello" : "Howdy"
+    }
+    
+    var audience = function () {
+        return flip(.5) ? "World" : "Universe"
+    }
+    
+    var phrase = greeting() + ", " + audience() + "!"
+    
+    return phrase
```

Comment:
As you probably know, we automatically `return` the last expression in the program. I wonder if we should introduce that idea here by dropping the `phrase` variable?